tools/license2rtf.js

```diff
@@ -2,112 +2,105 @@
 
 const assert = require('assert');
 const Stream = require('stream');
-const inherits = require('util').inherits;
 
 
 /*
  * This filter consumes a stream of characters and emits one string per line.
  */
-function LineSplitter() {
-  const self = this;
-  var buffer = '';
-
-  Stream.call(this);
-  this.writable = true;
+class LineSplitter extends Stream {
```

Comment:
Why not inherit from [`Stream.Writable`](https://nodejs.org/api/stream.html#stream_implementing_a_writable_stream) which provides automatic backpressure?

Reply:
Good point, I mostly just rewrote the classes without changing any semantics / inheritance.